__tests__/components/App.test.js

```diff
@@ -20,63 +24,70 @@ describe('<App />', () =>{
     expect(wrapper.find('div#app').length).toEqual(1)
   })
 
-  it('renders <Footer />', () => {
-    expect(wrapper.find(Footer).length).toBe(1)
+  it('renders the LoginPanel component', () => {
+    expect(wrapper.find(LoginPanel).length).toEqual(1)
   })
 
+  it('renders the Footer component', () => {
+    expect(wrapper.find(Footer).length).toBe(1)
+  })
 })
 
 describe("#routes", () => {
+  // pattern cribbed/adapted from https://stackoverflow.com/a/54807560
```

Comment:
an alternative might be to use https://redux.js.org/api/createstore.  i just sort of happened on this solution first, and it works well and feels intelligible/reasonable.  totally open to using `createStore` or whatever.

Reply:
`/shrug` works for me